.travis.yml

```diff
@@ -1,12 +1,9 @@
 sudo: false
 language: erlang
-before_script:
-    - wget https://github.com/erlang/rebar3/releases/download/3.6.1/rebar3
-    - chmod +x rebar3
 script:
-    - ./rebar3 eunit
-    - ./rebar3 ct
```

Comment:
Travis CI provides too old rebar3 version. That's why we need to manually download 3.6.1 and use it us before. The old version doesn't work with Erlang/OTP 21 correctly. @bryanhuntesl  please revert this and above changes.

Reply:
@michalwski - 

fixed by  - https://github.com/esl/exometer_core/pull/6/commits/111b828bb6c79debf58756e992de1775ed7f8505

pinged travis existing request to upgrade rebar3 - https://github.com/travis-ci/travis-ci/issues/9830#issuecomment-418301427

Reply:
Good to go ?

Reply:
Yes, I'm merging it.

Reply:
Thanks @michalwski